kurlkinds/pkg/lint/rego/vars.rego

```diff
@@ -0,0 +1,585 @@
+package kurl.installer
+
+# known_versions holds all addons and their respective versions, latest
+# is kept on its own variable.
+default known_versions = {
```

Comment:
is there some way to auto-generate this? i think we will also need to integrate this with the automated add-on version updates.

Reply:
an alternative and i think what i would prefer is to query the [api](https://k8s.kurl.sh/installer) with `http.send` https://www.openpolicyagent.org/docs/latest/policy-reference/#http

Reply:
I was seeing this repo (with the changes) as the single source of truth and everything else would be, eventually, migrated to start using it (including the `/installer` endpoint). Wasn't this the idea ? I may have got it wrong.

As for doing an external request that would be fine but we are planning to use this validation also during the host-preflight checks and that would not work in an air-gapped environment (ie. we gonna need to think in a solution to make this work without internet connection as well).